e2e/cypress/integration/messaging/system_message_spec.js

```diff
@@ -60,7 +60,7 @@ describe('System Message', () => {
         });
     });
 
-    it('MM-T213 System message limited options', () => {
+    it('MM-T426 System messages wrap properly', () => {
```

Comment:
I mistakenly update this per my last PR - https://github.com/mattermost/mattermost-webapp/pull/6130/files#diff-41c2ea6f67441c0c6de6e59cd495b80cR63